guides/json.md

```diff
@@ -6,61 +6,75 @@
 
 # Making Raw JSON REST Requests
 
-The OpenSearch client implements many high-level REST DSLs that invoke OpenSearch APIs. However you may find yourself in a situation that requires you to invoke an API that is not supported by the client. Use `client.transport.perform_request` to do so. See [samples/json](../samples/json) for a complete working sample.
+The OpenSearch client implements many high-level REST DSLs that invoke OpenSearch APIs. However you may find yourself in a situation that requires you to invoke an API that is not supported by the client. Use `client.http.get`, `head` , `put`, `post`, and `delete` in older versions to do so. See [samples/json](../samples/json) for a complete working sample.
```

Comment:
Use `client.transport.perform_request` in older versions (<= 2.3.x) and `client.http.get` and others in newer versions?

Reply:
I think "in older versions" would be removed from this line: "Use `client.http.get`, `head` , `put`, `post`, and `delete` to do so"?

Reply:
Good eyes, fixed.